docs/content/integrations/airbyte-cloud.mdx

```diff
@@ -0,0 +1,234 @@
+)
+```
+
+#### Adding a resource
+
+Manually built Airbyte assets require an `airbyte_resource`, which defines how to connect and interact with your Airbyte Cloud instance.
+
+We can add the Airbyte resource we configured above to our Airbyte assets by doing the following:
+
+```python startafter=start_airbyte_manual_config endbefore=end_airbyte_manual_config file=/integrations/airbyte/airbyte.py dedent=4
+from dagster_airbyte import build_airbyte_assets
+
+from dagster import with_resources
+
+airbyte_assets = with_resources(
+    build_airbyte_assets(
+        connection_id="87b7fe85-a22c-420e-8d74-b30e7ede77df",
+        destination_tables=["releases", "tags", "teams"],
+    ),
+    # Use the airbyte_instance resource we defined in Step 1
+    {"airbyte": airbyte_instance},
+)
+```
+
+---
+
+## Step 3: Adding downstream assets
+
+<Note>
+  Looking to orchestrate Airbyte with dbt? Check out our{" "}
+  <a href="https://github.com/dagster-io/dagster/tree/master/examples/assets_modern_data_stack">
+    Modern Data Stack example
+  </a>{" "}
+  and our <a href="/integrations/dbt">dbt integration docs</a>.
+</Note>
+
+Once you have loaded your Airbyte Cloud assets into Dagster, you can create assets which depend on them. These can be other assets pulled in from external sources such as [dbt](/integrations/dbt) or assets defined in Python code.
+
+In this case, we have an Airbyte connection which is storing data in the `stargazers` table in our Snowflake warehouse. We specify the output [IO manager](/concepts/io-management/io-managers) to tell downstream assets how to retreive the data.
+
+```python startafter=start_add_downstream_assets endbefore=end_add_downstream_assets file=/integrations/airbyte/airbyte.py dedent=4
+import json
+from dagster import asset, Definitions
+from dagster_airbyte import load_assets_from_airbyte_instance
+
+airbyte_assets = load_assets_from_airbyte_instance(
+    airbyte_instance,
+    io_manager_key="snowflake_io_manager",
+)
+
+@asset
+def stargazers_file(stargazers):
+    with open("stargazers.json", "w", encoding="utf8") as f:
+        f.write(json.dumps(stargazers, indent=2))
+
+defs = Definitions(
+    assets=[airbyte_assets, stargazers_file],
+    resources={"snowflake_io_manager": snowflake_io_manager},
+)
+```
+
+---
+
+## Step 4: Scheduling Airbyte Cloud syncs
+
+Once you have Airbyte Cloud assets, you can define a job that runs some or all of these assets on a schedule, triggering the underlying Airbyte sync.
+
+```python startafter=start_schedule_assets endbefore=end_schedule_assets file=/integrations/airbyte/airbyte.py dedent=4
+from dagster import (
+    ScheduleDefinition,
+    define_asset_job,
+    AssetSelection,
+    Definitions,
+)
+
+# materialize all assets
+run_everything_job = define_asset_job("run_everything", selection="*")
+
+# only run my_airbyte_connection and downstream assets
+my_etl_job = define_asset_job(
+    "my_etl_job", AssetSelection.groups("my_airbyte_connection").downstream()
```

Comment:
it is unclear in the guide so far where the group "my_airbyte_connection" comes from and might be worth highlighting how you properly select airbyte assets